plugins/Eq/EqSpectrumView.h

```diff
@@ -45,27 +47,43 @@ class EqAnalyser
 	float m_energy;
 	int m_sr;
 	bool m_active;
+	float m_sum;
+
 
 
 	EqAnalyser() :
```

Comment:
I fail to see why there is no `EqSpectrumView.cpp` with all these implementations.